frontend/src/main/pages/EnrollmentData/EnrollmentDataGraphPage.js

```diff
@@ -0,0 +1,15 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+
+// Stryker disable all : placeholder for future implementation
```

Comment:
Here's its ok, because the content of this page is going to be altered before this page actually shows up in the app.  But the other component is close to in it's final form.